package.json

```diff
@@ -6,27 +6,49 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test",
+    "lint": "ng lint"
   },
   "private": true,
   "dependencies": {
     "@angular/animations": "^14.2.0",
+    "@angular/cdk": "^14.0.0",
     "@angular/common": "^14.2.0",
     "@angular/compiler": "^14.2.0",
     "@angular/core": "^14.2.0",
     "@angular/forms": "^14.2.0",
     "@angular/platform-browser": "^14.2.0",
     "@angular/platform-browser-dynamic": "^14.2.0",
     "@angular/router": "^14.2.0",
+    "@blackbaud/sky-addin-client": "^1.1.5",
```

Comment:
Could be wrong, but I don't believe you need to include `@blackbaud/sky-addin-client` as a separate package these days.  I believe `skyux-lib-addin-client` will pull the dependency in automatically.

Reply:
Good catch, you're correct. I've updated to remove this reference